index.html

```diff
@@ -173,126 +173,445 @@ <h2>
     </section>
     <section>
       <h2>
-        <code>pointerlockchange</code> and <code>pointerlockerror</code> Events
+        Pointer Lock and Interfaces
       </h2>
-      <p>
-        Two events are used to communicate pointer lock state change or an
-        error in changing state. They are named <dfn>pointerlockchange</dfn>
-        and <dfn>pointerlockerror</dfn>. If pointer lock is entered or exited
-        for any reason a <a>pointerlockchange</a> event must be sent.
-      </p>
-      <p>
-        <a>User agents</a> must deliver these events by <a data-lt=
-        "queue a task">queuing a task</a> to <a>fire an event</a> of the
-        appropriate name with its <code>bubbles</code> attribute [[DOM]] set to
-        false to the pointer lock <a>target</a> element's [=Node/node
-        document=].
-      </p>
-      <p class="note">
-        Magnification software increases the size of content on the screen. It
-        uses the mouse to move the magnified point of focus around. When a
-        pointer lock is initiated, the magnification software needs to switch
-        to using the keyboard to move the magnified point of focus around
-        instead. When a pointerlockchange event is fired, web browsers
-        therefore need to make sure the event is communicated to assistive
-        technologies like screen magnifiers.
-      </p>
+      <section>
+        <h3>
+          The <dfn>pointer lock state</dfn> definition
+        </h3>
+          <p><a>Pointer lock state</a> is the state where a single DOM element, which
+            we will call the <dfn>pointer-lock target</dfn>, receives all mouse events and the cursor
+            is hidden.</p>
+          <p>
+            Once in the <a>pointer lock state</a> the <a>user agent</a> has a
+            <a>pointer-lock target</a>, a <dfn>pointer-lock options</dfn>, which is a <a>PointerLockOptions</a> and a
+            <dfn>cursor position</dfn> which is a pair of numbers representing the location of the system mouse
+            cursor when the Pointer Lock State was entered (the same location that is reported in <code>screenX</code>,
+            <code>screenY</code>). The <a>pointer-lock target</a> must receive all relevant user generated {{MouseEvent}}
+            events: namely, all user-generated `mousemove`, `mousedown`, `mouseup`, `click`, `dblclick`,
+            `auxclick`, and `wheel` [[ui-events]]. No other elements may receive these events while in
+            <a>pointer lock state</a>. There will be no dispatching of events that require the concept of a mouse cursor: namely,
+            `mouseenter`, `mouseleave`, `mouseover`, `mouseout`, `drag`, and `drop`.
+          </p>
+          <p>
+            <dl data-dfn-for="PointerLockOptions" data-link-for=
+            "PointerLockOptions">
+              While in the <a>pointer lock state</a> if the <a>pointer-lock options</a>'
+              <a>unadjustedMovement</a> member is <code>true</code> the event 
```

Comment:
When linking to WebIDL things, it's best to use the special WebIDL syntax.

```suggestion
              {{PointerLockOptions/unadjustedMovement}} member is `true` the event 
```

Please see our [WebIDL shorthands cheat sheet](https://respec.org/docs/#Shorthands-Guide). 

Be great to update the changes throughout this pull request 🙏

Reply:
Done